src/app/visual/fonts/fonts.component.scss

```diff
@@ -0,0 +1,19 @@
+.icon::before {
+  display: inline-block;
+  font-style: normal;
+  font-variant: normal;
+  text-rendering: auto;
+  -webkit-font-smoothing: antialiased;
+}
+
+.login::before {
+  font-family: "Font Awesome 5 Free"; font-weight: 900; content: "\f007";
```

Comment:
Is the point of this line to "prove" that FontAwesome 5 is registered on the page in all of its forms?

Just curious what this serves in relation to a consumer (they wouldn't be writing "Font Awesome 5 Free", correct)?

Reply:
The point of these custom classes are to show that the pseudo-class technique still works for V4 and V5 icons. It is mostly taken directly from the migration guide as something you'll have to fix from V4 -> V5. https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4